### ISSUE TITLE
Copy session need input username/password again

### ISSUE THREAD

**Issue by martin-chips** [kingToolbox/WindTerm#906]
hi, 
We have many servers with the same username/password, so I need input username/password again and again when using WindTerm.
Can you help to consider that  add the function copy new session with the orginal uername/password?

Thank you very much.

**Comment by yuanyongfei** [comment]
I have the same problem, I don't know how to make it, wait for the official response, this is very unfriendly

**Comment by wxjback** [comment]
yes, I lost my password either, after I use the copy of old profile folder. waiting for solution.

**Comment by martin-chips** [comment]
hi, @kingToolbox , Can you help to answer this question?

**Comment by yuanyongfei** [comment]
我的解决了, 
![image](https://user-images.githubusercontent.com/109744745/182819101-f5b122bf-cd58-4410-9d1a-b5d492782c36.png)
